### DISCUSSION TITLE
Installation Error

### DISCUSSION THREAD

**Discussion by kayojo** [nguyen-group/QERaman#15] [Q&A]
Hello, I need help with my installation. After I installed QE version 7.2 via Cmake, I encountered an error during the installation process of QERaman. Specifically, I received the following error message:

supp.f90:10:6:

   10 |   USE kinds, ONLY : DP
      |      1
Fatal Error: Cannot open module file ‘kinds.mod’ for reading at (1): No such file or directory
compilation terminated.
../../make.inc:16: recipe for target 'supp.o' failed
make: *** [supp.o] Error 1
what can I do to correct this, note I successfully installed QE 7.2, and it is running calculations. I have also attached the error message here. Thanks for your help
[error file.txt](https://github.com/nguyen-group/QERaman/files/14823912/error.file.txt)

**Comment by kayojo** [comment]
Thanks, I solved this by re_installing QE and setting some missing libraries in my environment variables

**Reply by nguyen-group** [reply]
Sorry for late response. It is good to hear that.
